l10n/templates/strings.properties

```diff
@@ -136,6 +141,7 @@ edit_topsites_unpin_button=Unpin this site
 edit_topsites_edit_button=Edit this site
 edit_topsites_dismiss_button=Dismiss this site
 edit_topsites_add_button=Add
+edit_topsites_add_button_label=Add Top Site
```

Comment:
I'm not sure if the screenshot is confusing me: this looks more like a tooltip than a label?

Reply:
Oh it is indeed a tooltip. I don't think any of the strings have used `_tooltip` although we can start now… ?

Reply:
Yes, I think that would help. Typically label = button label (imperative), tooltip can be descriptive in some languages.

Reply:
Okay, fixing up in https://github.com/mozilla/activity-stream/pull/3956 then merging